frontend/src/routes/users/[id]/+page.svelte

```diff
@@ -77,15 +78,16 @@
                   <dt class="text-sm font-medium text-gray-500">Phone</dt>
                   <dd class="mt-1 text-sm text-gray-900">+1 555-555-5555</dd>
                 </div>
-                <div class="sm:col-span-2">
-                  <dt class="text-sm font-medium text-gray-500">About</dt>
-                  <dd class="mt-1 text-sm text-gray-900">
-                    Fugiat ipsum ipsum deserunt culpa aute sint do nostrud anim incididunt cillum
-                    culpa consequat. Excepteur qui ipsum aliquip consequat sint. Sit id mollit nulla
-                    mollit nostrud in ea officia proident. Irure nostrud pariatur mollit ad
-                    adipisicing reprehenderit deserunt qui eu.
-                  </dd>
-                </div>
+                {#if about_info !== '' && about_info !== null}
```

Comment:
Both `''` and `null` in js are `false` so you could just use `if (about_info)`

![image](https://user-images.githubusercontent.com/5938539/229562133-93246736-f3dc-4fb4-8f89-4226d8478363.png)